content/concepts/pam.md

```diff
@@ -145,7 +145,7 @@ Below is the JSON standard format representing asset categories. Notice how it's
 :::
 
 ::: warning
-This resource is **deprecated** and is removed from the API since the 4.0. As a result, from now on, all the endpoints regarding this resource are deprecated. To understand why, we recommend you to read this [Medium post](https://medium.com/akeneo-labs/between-stability-and-innovation-c2d2dd61a804), we wrote on this special occasion.  
+This resource is **deprecated** and is removed from the REST  since the 4.0. As a result, from now on, all the endpoints regarding this resource are deprecated. To understand why, we recommend you to read this [Medium post](https://medium.com/akeneo-labs/between-stability-and-innovation-c2d2dd61a804), we wrote on this special occasion.  
```

Comment:
```suggestion
This resource is **deprecated** and is removed from the REST API since 4.0. As a result, from now on, all the endpoints regarding this resource are deprecated. To understand why we recommend you to read this [Medium post](https://medium.com/akeneo-labs/between-stability-and-innovation-c2d2dd61a804), we wrote on this special occasion.  
```